src/main/java/eu/greev/dcbot/ticketsystem/service/TicketService.java

```diff
@@ -156,6 +165,8 @@ public boolean claim(Ticket ticket, User supporter) {
         if (!ticket.getInfo().equals(Strings.EMPTY))
             builder.addField("Information", ticket.getInfo(), false);
 
+        ticket.getChannel().getThreadChannels().get(0).addThreadMember(supporter).queue();
```

Comment:
Okay for now, but eventually, we probably want to do something with the channel id instead of gambling that the first channel will always be the correct one